crates/holochain_integrity_types/src/dna_modifiers.rs

```diff
@@ -103,3 +105,17 @@ pub trait TryFromDnaProperties {
     where
         Self: Sized;
 }
+
+#[cfg(feature = "schema")]
+fn properties_schema(_: &mut schemars::SchemaGenerator) -> schemars::schema::Schema {
```

Comment:
This is only here for the purpose of json schema generation and does not affect the format of properties that can actually be used in practice, right? I've been assuming so far that properties can be of arbitrary, including non-JSON format.

Reply:
The value isn't really constrained here because of the generic `<P>` used in `DnaModifiersOpt`. But elsewhere we specify that it should be treated as a YAML properties. So if this value is present, I think it would ideally deserialize to a YAML object. Beyond that, we aren't placing any constraints on the contents of the properties no.

So I think the answer to your question is: correct, and we're just making a suggestion in the schema that will help people write properties that can be fetched easily inside the zome using the YAML helpers.